charts/aws-ssm/aws-ssm-chaos-by-id/experiment.yaml

```diff
@@ -42,7 +42,7 @@ spec:
       - apiGroups: ["litmuschaos.io"]
         resources: ["chaosengines","chaosexperiments","chaosresults"]
         verbs: ["create","list","get","patch","update","delete"]
-    image: "litmuschaos/go-runner:ci"
+    image: "litmuschaos/go-runner:latest"
```

Comment:
If it's on `m-agent` branch let's give him the m-agent tag only.

Reply:
This is the aws-ssm experiment chart, not the process-kill experiment chart. The process-kill experiment image has been tagged with `m-agent` tag only.

Reply:
I see